src/boot/parser.mly

```diff
@@ -289,6 +289,8 @@ atom:
   | LPAREN seq RPAREN
       { if List.length $2 = 1 then List.hd $2
         else tuple2record (mkinfo $1.i $3.i) $2 }
+  | LPAREN mexpr COMMA RPAREN
```

Comment:
I don't see a corresponding change for `pprint`, how are singleton tuples printed there?

Reply:
Good catch! We updated the pretty printer now.